napari/utils/colormaps/_tests/test_colormap.py

```diff
@@ -43,7 +43,10 @@ def test_binned_colormap_with_control_points():
     """Test a binned with control points."""
     colors = np.array([[0, 0, 0, 1], [0, 1, 0, 1], [0, 0, 1, 1]])
     cmap = Colormap(
-        colors, name='testing', interpolation='zero', controls=[0, 0.2, 0.3, 1]
+        colors=colors,
```

Comment:
Do you want to revert this now that (as I read the conversation) you've re-added positional args to Colormap? It'd be nice to keep testing that that works.

Reply:
done, i reverted most tests, but missed this one